Mail/Views/MoveEmailView.swift

```diff
@@ -25,41 +25,56 @@ import RealmSwift
 import SwiftUI
 
 struct MoveEmailView: View {
-    @LazyInjectService private var matomo: MatomoUtils
+    typealias MoveHandler = (Folder) -> Void
 
-    @Environment(\.dismissModal) var dismissModal
+    @LazyInjectService private var matomo: MatomoUtils
 
+    @Environment(\.dismissModal) private var dismissModal
     @EnvironmentObject private var actionsManager: ActionsManager
 
-    typealias MoveHandler = (Folder) -> Void
-
     @ObservedResults(Folder.self, where: { $0.role != .draft && $0.toolType == nil }) var folders
```

Comment:
It looks fine for a move view, but if the view is a bit unresponsive I would suggest to add a viewModel.

We could then observe the Folders with realm in a serial queue (not main) an process the various properties (filteredFolders / roleFolders / userFolders) asyncronously.

Reply:
Yeah that's a good point actually 👍